data/remnant missions.txt

```diff
@@ -576,7 +576,8 @@ mission "Remnant: Technology Available"
 	on offer
 		payment 2000000
 		set "license: Remnant"
-		log "The prefects have reached a consensus that you should be considered a valuable friend of the Remnant, and shall be given permission to purchase their ships and technology."
+		log "The prefects have reached a consensus: Shall be considered a valuable friend of the Remnant, and given permission to purchase their ships and technology."
```

Comment:
```suggestion
		log "The prefects have reached a consensus: have been considered a valuable friend of the Remnant, and was given permission to purchase their ships and technology."
```

Reply:
That sounds good, but "have been considered" sounds like <first> was a valuable friend, past tense. As in, no longer is a one.

Reply:
"Shall be" is future tense though, when the truth is that you're considered a valuable friend now. And I don't necessarily see "have been" implying no longer. Putting the pronouns in there, it would read "I have been considered a valuable friend," which is different from "I was considered a valuable friend." Both are past tense, but one implying no longer ("was") while the other does not.

Reply:
`Am now`?

Reply:
I prefer `Am now considered a valuable friend" as that clearly indicates the log is talking about a present state of affairs.

Reply:
Simply "now considered" (drop the "am") would flow better.

Reply:
Agreed. I'll make the change as soon as I get Git setup on my machine again.

Reply:
Changed